### PR TITLE
[Release] 1.13.0

### PR DESCRIPTION
**Pull request by olivernybroe**

### CODE REVIEW

File: tests/Infrastructure/Repositories/LocalFilesRepositoryTest.php [hunk 1]
```diff
@@ -79,6 +79,12 @@ public function testPassFileInsteadOfDirectory(): void
 
         self::assertCount(1, $files);
         self::assertInstanceOf(SplFileInfo::class, $files[0]);
-        self::assertStringContainsString('/Fixtures/FileToInspect.php', $files[0]->getRealPath());
+        $path = $files[0]->getRealPath();
+
+        if ($path === false) {
+            self::fail("Path cannot be false.");
```
Comment (Jibbarth) [inline tests/Infrastructure/Repositories/LocalFilesRepositoryTest.php]: Didn't know about this one :+1:

Comment (olivernybroe) [reply tests/Infrastructure/Repositories/LocalFilesRepositoryTest.php]: Apparently phpstan wouldn't let it pass without checking this. Weird that it haven't failed before now though. 😕

Comment (Jibbarth) [reply tests/Infrastructure/Repositories/LocalFilesRepositoryTest.php]: A new phpstan version has been released this night. That might explain this :wink:

Comment (olivernybroe) [reply tests/Infrastructure/Repositories/LocalFilesRepositoryTest.php]: Ah nice catch, must be it. 😆

**Pull request merged**